media_driver/linux/gen11/ddi/media_libva_caps_g11.cpp

```diff
@@ -410,6 +410,68 @@ VAStatus MediaLibvaCapsG11::GetPlatformSpecificAttrib(VAProfile profile,
             }
             break;
         }
+#if VA_CHECK_VERSION(1, 12, 0)
+        case VAConfigAttribEncHEVCFeatures:
+        {
```

Comment:
VAConfigAttribEncHEVCBlockSizes was separated for VAEntrypointEncSliceLP & VAEntrypointEncSlice. 
but  VAConfigAttribEncHEVCFeatures not, in my impression, they also have different features. @Tianhaol  could you help to check it?

Reply:
@thomasli21801

Reply:
Yes, for VAConfigAttribEncHEVCFeatures, implement it as one if VAEntrypointEncSliceLP and  VAEntrypointEncSlice have the same values for each field.

Reply:
Block size is related w/ log2_max_coding_tree_block_size_minus3 below for different implementations. Others LGTM.